ProcessLib/ThermoHydroMechanics/CreateThermoHydroMechanicsProcess.cpp

```diff
@@ -0,0 +1,297 @@
+
+    // Storage coefficient
+    auto& storage_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_storage_coefficient}
+        "storage_coefficient", parameters, 1);
+
+    DBUG("Use \'%s\' as storage coefficient parameter.",
+         storage_coefficient.name.c_str());
+
+    // Fluid viscosity
+    auto& fluid_viscosity = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_fluid_viscosity}
+        "fluid_viscosity", parameters, 1);
+    DBUG("Use \'%s\' as fluid viscosity parameter.",
+         fluid_viscosity.name.c_str());
+
+    // Biot coefficient
+    auto& biot_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_biot_coefficient}
+        "biot_coefficient", parameters, 1);
+    DBUG("Use \'%s\' as Biot coefficient parameter.",
+         biot_coefficient.name.c_str());
+
+    // Porosity
+    auto& porosity = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_porosity}
+        "porosity", parameters, 1);
+    DBUG("Use \'%s\' as porosity parameter.", porosity.name.c_str());
+
+    // Solid density
+    auto& solid_density = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_solid_density}
+        "solid_density", parameters, 1);
+    DBUG("Use \'%s\' as solid density parameter.", solid_density.name.c_str());
+
+    // Fluid density
+    auto const& fluid_density = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_fluid_density}
+        "fluid_density", parameters, 1);
+    DBUG("Use \'%s\' as fluid density parameter.", fluid_density.name.c_str());
+
+    // linear thermal expansion coefficient for solid
+    auto const& solid_linear_thermal_expansion_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_solid_linear_thermal_expansion_coefficient}
+        "solid_linear_thermal_expansion_coefficient", parameters, 1);
+    DBUG("Use \'%s\' as solid linear thermal expansion coefficient parameter.",
+         solid_linear_thermal_expansion_coefficient.name.c_str());
+
+    // volumetric thermal expansion coefficient for fluid
+    auto const& fluid_volumetric_thermal_expansion_coefficient = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__THERMO_HYDRO_MECHANICS_fluid_volumetric_thermal_expansion_coefficient}
+        "fluid_volumetric_thermal_expansion_coefficient", parameters, 1);
+    DBUG("Use \'%s\' as fluid volumetric thermal expansion coefficient parameter.",
+         fluid_volumetric_thermal_expansion_coefficient.name.c_str());
+
```

Comment:
how about using fluid props here? Then different density models automatically yield different thermal expansivities and fluid compressibilities?